packages/manager/modules/pci/src/projects/project/private-networks/onboarding/onboarding.controller.js

```diff
@@ -0,0 +1,42 @@
+import reduce from 'lodash/reduce';
+import illustration from './assets/private-networks.png';
+import { GUIDES } from './onboarding.constants';
+
+export default class PciPrivateNetworkEmptyController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.loading = false;
+    this.illustration = illustration;
+    this.onboardingTrackPrefix = 'onboarding::';
+    this.guides = reduce(
```

Comment:
**suggestion:** `Array.prototype.reduce` can be used.

Reply:
yes, we can use. But we had used loadash at almost everywhere so thought to use the same.